plugins/storage/volume/storpool/src/main/java/org/apache/cloudstack/storage/snapshot/StorPoolConfigurationManager.java

```diff
@@ -44,13 +44,23 @@ public class StorPoolConfigurationManager implements Configurable {
             "The interval in seconds to get StorPool template statistics",
             false);
 
+    public static final ConfigKey<Integer> DeleteAfterInterval = new ConfigKey<>("Advanced", Integer.class,
+            "storpool.delete.after.interval", "0",
+            "The interval (in seconds) after the StorPool snapshot will be deleted",
+            false);
+
+    public static final ConfigKey<Integer> ListSnapshotsWithDeleteAfterInterval = new ConfigKey<>("Advanced", Integer.class,
+            "storpool.list.snapshots.delete.after.interval", "360",
+            "The interval (in seconds) to fetch the StorPool snapshots with deleteAfter flag",
+            false);
```

Comment:
can keep these configs with StoragePool scope, so that these can be changed for other storpool storage if needed.

Reply:
Thanks, I made it for the storage scope